### ISSUE TITLE
Enable Add Student function to admin/student

### ISSUE THREAD

**Issue by altermattw** [skiadas/HanoverEvals#139]
It's currently possible to add an existing student to a course under the Roster page.  However, there is not a way to add a student who does not yet exist (to "create" a student).

**Comment by skiadas** [comment]
Is there a use case for that? All HC students and instructors would be loaded automatically by the system.

**Comment by altermattw** [comment]
I don't think it would be used very often.  But if there is some kind of error made by Jenzabar, it would be nice to be able to manually add a student.  Last spring, one of my students disappeared from my my.hanover roster for 3 days, during which he couldn't access his email or anything on moodle.

**Comment by altermattw** [comment]
If we decide to create an Add Student feature, we should also create an Add Instructor function for admin/instructor.

**Comment by skiadas** [comment]
I will create a form for adding a new "user" in general, and an option for the kind of user to create (student/instructor/admin).

**Comment by skiadas** [comment]
The more I think about it, this might need to be combined with a move to having one "user" table, and a separate table assigning roles to users. It would make more sense, and would also allow what is now impossible, namely having an instructor who is also an admin.

Not sure how the interface should work though for someone who has more than one role. Perhaps they should somehow be logged in as one "role", and be allowed to change that role.

I think before we spend more time implementing this, we should find out if it is something that is really needed. It complicates the design for what might simply be an imagined need.

I think I'll remove it from "stable release 1" for now. I did prepare a rudimentary commit for it, but it's pretty ugly right now. I'll throw it in anyway, but I'll also disable the menu option.

**Comment by skiadas** [comment]
See 3c080c638894c6d2fb375cd6653585d97cb4bbc7
